packages/perseus/src/widgets/radio/choice-icon/choice-icon.jsx

```diff
@@ -1,87 +1,231 @@
 // @flow
-/* Component for rendering a letter icon in radio choice */
+/* Component for rendering a letter icon in a library radio choice */
 
 import Color from "@khanacademy/wonder-blocks-color";
+import {StyleSheet, css} from "aphrodite";
 import * as React from "react";
 
+import InlineIcon from "../../../components/inline-icon.jsx";
+import {iconCheck, iconMinus} from "../../../icon-paths.js";
+import * as styleConstants from "../../../styles/constants.js";
+import FocusRing from "../focus-ring.jsx";
 import {getChoiceLetter} from "../util.js";
 
-import LibraryChoiceIcon from "./library-choice-icon.jsx";
-import SATChoiceIcon from "./sat-choice-icon.jsx";
+import CrossOutLine from "./cross-out-line.jsx";
+import sharedStyles, {LIBRARY_ICON_SIZE} from "./shared-styles.js";
 
-type ChoiceIconProps = {|
+type ChoiceIconProps = {
     pos: number,
     checked: boolean,
     crossedOut: boolean,
     focused: boolean,
     hovered: boolean,
     pressed: boolean,
-    correct: boolean,
+    correct: ?boolean,
     showCorrectness: boolean,
-    // TODO(amy): if we go this "product" flag route, define this type
-    // somewhere shared
-    product: "sat" | "library",
     multipleSelect: boolean,
     reviewMode: boolean,
+    primaryProductColor?: string,
     previouslyAnswered: boolean,
     // TODO(mdr): The CrossOutButton needs a transparent-background ChoiceIcon,
     //     so I've added this prop. I'm not sure why we have backgrounds in the
     //     general case, though? When does the choice container have a
     //     non-white background, aside from SAT, which uses a different icon?
     transparentBackground?: boolean,
+};
 
-    primaryProductColor?: string,
-|};
+function ChoiceInner(props: {
```

Comment:
This is all moved from `library-choice-icon.jsx`.  Because we don't have two types of choice icon anymore, I merged them together. :)